opengrok-web/src/main/webapp/js/utils-0.0.29.js

```diff
@@ -1784,7 +1784,7 @@ function initMinisearchAutocomplete(config) {
 }
 
 function initAutocompleteForField(inputId, field, config, dataFunction, errorElemId) {
-    if (config.allowedFields && !config.allowedFields.includes(field)) {
+    if (config.allowedFields && config.allowedFields.indexOf(field)<0) {
```

Comment:
there should be a space `field')<0)` -> `field') < 0)`

Reply:
thx, fixed